go/common/identity/identity.go

```diff
@@ -35,6 +37,13 @@ const (
 	tlsKeyFilename  = "tls_identity.pem"
 	tlsCertFilename = "tls_identity_cert.pem"
 
+	tlsEphemeralKeyBaseFilename  = "tls_ephemeral"
+	tlsEphemeralCertBaseFilename = "tls_ephemeral_cert"
+
+	tlsEphemeralGenCurrent     = ""
```

Comment:
Hm, could also just serialize the keys and store them in the localstorage badger db? I think it would simplify some of the shuffling as the rotation could be done in a transaction. 
_I feel_ it would also be a bit nicer as the keys would be a bit more hidden from the user (which i think makes sense, as these should be ephemeral).

Am also fine with this if it stays as its.

Reply:
Given that we are storing all the other keys in PEM files in the data directory I think it makes it more consistent to store these in the same way. It is true that these are ephemeral though.